pkg/component/runtime/state.go

```diff
@@ -131,15 +134,6 @@ func (s *ComponentState) Copy() (c ComponentState) {
 	return c
 }
 
-func (s *ComponentState) syncComponent(comp *component.Component) bool {
```

Comment:
Moving this inline since it only had one caller and its return value was ignored